_elm_package.sh

```diff
@@ -95,8 +95,31 @@ _suggest_package_publish_flags()
     COMPREPLY=( $(compgen -W "${flag_options}" -- $cur) )
 }
 
+_is_suggesting_package_name()
+{
+    local flags_before_package_name collect_flags
+
+    flags_before_package_name=0
+    collect_flags=1
```

Comment:
Why is this needed?

Reply:
There are two situations
• When flags are before package name `elm-package install --flag1 --flag2 ... package/name`
• When flags are after package name `elm-package install package/name --flag1 --flag2 ...`

We want to count flags which are before package name to exclude them. So that package name will have always the same index. Which equals to `2`.